pyproject.toml

```diff
@@ -43,7 +43,7 @@ numba = "^0.57.0"
 eko = "^0.13.4"
 pandas = "^1.3.0"
 rich = "^12.4.4"
-adani = "^0.13"
+adani = "^0.14"
```

Comment:
Just to make you know: I've also pushed 0.14.1 that has also mac wheels

Reply:
Thanks! Updated mine [68eb5ee](https://github.com/NNPDF/yadism/pull/242/commits/68eb5ee318cc9fc2ff986043f7f307dac38a6551) and everything still works fine, so this is very good.

Reply:
do you want to pin v0.14.1? i.e. putting it here?